### DISCUSSION TITLE
Does it support CosmosDB as Storage Provider ? If not, any future plans.

### DISCUSSION THREAD

**Discussion by apandey846** [jobrunr/jobrunr#309] [General]
These days CosmosDb's is being widely used for NoSql DB. Was wondering if it support as could not find under the StorageProvider implementation section or any plans to support ?

**Comment by rdehuyss** [comment]
There were already quite some questions about CosmosDB (see the issues, ... ). I'm open for a PR but don't plan to add it myself at this point in time.

**Reply by rdehuyss** [reply]
Although I just read that it has an API which can be used with the MongoDB driver. Did you try that already?
